Panto/css/style.css

```diff
@@ -50,18 +54,97 @@
 .hero {
   text-align: center;
   color: var(--color-white);
+  padding-block: 16rem;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 4.4rem;
+}
+
+.hero-text-box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 2.4rem;
+}
+
+.hero-text-box h1 {
+  width: 80%;
+}
+
+.hero-description {
+  width: 45%;
```

Comment:
1- use logical properties like inline-size
2- What if your screen gets too tiny? I would use  something like `max-inline-size: 50rem` (we can try out different things, the thing that I just doesn't necessarily have to be true)